model/parVarSheet.cc

```diff
@@ -133,11 +133,11 @@ namespace minsky
 			    else
 			      {
                     cairo_move_to(cairo,x,y-1.5*rowHeight);
-                    pango.setMarkup(value->name+":");
+                    pango.setMarkup(latexToPango(value->name)+":");
                     pango.show();  					    
                     for (size_t k=0; k<rank-1; k++)  
                       {   
-                        y+=rowHeight; // allow room for header row
+                        if (k%2==0) y+=rowHeight; // allow room for header row
```

Comment:
Just so you know - % is a relatively expensive operation (think long division), and modulo powers of two can be done via much cheaper bit masks. x%2==0 is equivalent to (x&1)==0. Be careful with operator precedence though when using these operators - it tends to be counterintuitive, so liberal use of parentheses is advisable. 

I don't know how crucial performance is in this bit of code.

Reply:
Thanks, I've replaced the 3 instances I added with this suggestion. There's still room for improvement in this feature, especially for higher rank parameters, but it will have to wait until I've sorted out bug 1258 that Steve raised recently.